src/sentry/models/environment.py

```diff
@@ -48,8 +48,28 @@ def get_cache_key(cls, organization_id, name):
         return 'env:2:%s:%s' % (organization_id, md5_text(name).hexdigest())
 
     @classmethod
+    def get_name_or_default(cls, name):
+        return name or ''
```

Comment:
Not related to this but using an `or` here instead of `is not None` seems odd to me but that's how it is I guess